org-brain.el

```diff
@@ -3025,11 +3038,22 @@ LINK-TYPE will be \"brain\" by default."
                         (org-brain-title entry))
                 nil choice)
              (org-brain-add-resource
-              (concat "file:" (file-relative-name
+              (concatenate 'string
+			   "file:"
+			   (file-relative-name
                                (buffer-file-name)
-                               (file-name-directory (org-brain-entry-path choice))))
-              (concat (and (stringp org-brain-backlink) org-brain-backlink)
-                      (file-name-base))
+                               (file-name-directory (org-brain-entry-path choice)))
+			   (if (and org-brain-backlink-heading
+				     (ignore-errors (org-get-outline-path t)))
+				(concat "::* "
+					(nth 0 (last (org-get-outline-path t))))
+			      ))
```

Comment:
In emacs-lisp parentheses shouldn't be on a line of their own.

Reply:
I will fix it. Do you have any emacs packages suggestions for working with lisp?